common/app/components/App.tsx

```diff
@@ -426,7 +426,7 @@ function App({ origin, logoUrl, settings, extension, fetcher, onSettingsChanged,
                         newCard,
                         settingsRef.current.audioPaddingStart,
                         settingsRef.current.audioPaddingEnd,
-                        true
+                        settings.recordWithAudioPlayback
```

Comment:
Here, use settingsRef to access the setting.

Reply:
Fixed